policyengine_us/parameters/gov/states/nh/tax/income/rate.yaml

```diff
@@ -1,6 +1,6 @@
-description: New Hampshire taxes bonds, notes, and money at interest at this rate.
+description: New Hampshire taxes interest and dividends at this rate.
 values:
-  2022-01-01: 0.05
+  2021-01-01: 0.05
```

Comment:
Could you add a source? The legal code is ambiguous since the most recent action took effect in 2022.

Reply:
Done